tools/provision.py

```diff
@@ -207,6 +207,7 @@ def main(options):
     run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
     run(["tools/setup/emoji/build_emoji"])
     run(["scripts/setup/generate_secrets.py", "--development"])
+    run(["tools/update-authors-json", "--use-fixture"])
```

Comment:
Would moving or copying be better, if we just move it to a setup() method, wouldn't the data be unavailable if the user didn't run the tests.